tests/integration/test_pairwise.py

```diff
@@ -18,18 +18,132 @@
 class TestStandardizedResiduals(TestCase):
     """Test cr.cube implementation of column family pairwise comparisons"""
 
+    def test_mr_x_cat_residuals(self):
+        chisq_rows = [
```

Comment:
large expectations like this are better stored as JSON and loaded rather than inlining like this. It makes the test much harder to read when it's more than a screenful.